src/components/PreviewFrame.jsx

```diff
@@ -9,6 +9,7 @@ import {sourceDelimiter} from '../util/generatePreview';
 const sandboxOptions = [
   'allow-forms',
   'allow-popups',
+  'allow-same-origin',
```

Comment:
So yes, this is a problem. In particular, it presents a security vulnerability—a user could click a malicious gist import link, which contains JavaScript that breaks out of the iframe and attempts to hijack, for instance, the logged in GitHub credentials. The stakes probably aren’t _that_ high given the details of what we actually auth with GitHub etc., but I am a proponent of bright lines where this stuff is concerned.

What about this: when dragging is started, pop a transparent div into existence that overlays the preview frame. That should naturally catch the subsequent `mouseover` events. When dragging ends, of course, we’d make the frame disappear. Thoughts?